web/cypress/integration/route/create-edit-duplicate-delete-route.spec.js

```diff
@@ -163,12 +163,15 @@ context('Create and Delete Route', () => {
 
   it('should delete the route', function () {
     cy.visit('/routes/list');
-    [newName, duplicateNewName].forEach(function (routeName) {
-      cy.get(this.domSelector.nameSelector).type(routeName);
+    const { domSelector, data } = this;
+    const routeNames = [newName, duplicateNewName];
+    routeNames.forEach(function (routeName) {
+      cy.get(domSelector.nameSelector).type(routeName);
       cy.contains('Search').click();
       cy.contains(routeName).siblings().contains('Delete').click();
       cy.contains('button', 'Confirm').click();
-      cy.get(this.domSelector.notification).should('contain', this.data.deleteRouteSuccess);
-    }).bind(this);
+      cy.get(domSelector.notification).should('contain', data.deleteRouteSuccess);
```

Comment:
To make the test case stable, it would be better to close the notification here.